src/components/Module.jsx

```diff
@@ -0,0 +1,37 @@
+// ============================================================================================
+// IMPORTS
+// ============================================================================================
+
+import PropTypes from "prop-types";
+
+// ============================================================================================
+// COMPONENT DEFINITION
+// ============================================================================================
+
+/*********************************************************************************************/
+
+function Module({ moduleData }) {
+  return (
+    <>
+      <h2>{moduleData?.title}</h2>
+
+      <p>{moduleData?.description}</p>
+
```

Comment:
Used conditional rendering to render the link (<a>) only if url is present in moduleData.

 {url && (
        <p>
          <a href={url} target="_blank" rel="noreferrer">
            Show
          </a>
        </p>
      )}

Reply:
Agreed.